src/main.rs

```diff
@@ -183,7 +183,7 @@ fn main() -> std::io::Result<()> {
         .values_of("LOAD_PATH")
         .map_or_else(Vec::new, |vals| vals.map(Path::new).collect());
 
-    let style = match matches.value_of("STYLE").unwrap() {
+    let style = match &matches.value_of("STYLE").unwrap().to_lowercase() as &str {
```

Comment:
```suggestion
    let style = match matches.value_of("STYLE").unwrap().to_lowercase().as_str() {
```
probably would prefer this style, avoiding the cast, but not blocking